generated/global-autoconfig-properties/pom.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.google.cloud</groupId>
+        <artifactId>generated-parent</artifactId>
+        <version>3.5.0-SNAPSHOT</version>
+    </parent>
+
+
+    <artifactId>global-spring-autoconfig-properties</artifactId>
+    <name>Spring Cloud GCP - Global Autoconfig Properties</name>
+    <description>Common properties for generated spring autoconfigurations</description>
+
+    <properties>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+    </properties>
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter</artifactId>
+            <version>2.7.3</version>
```

Comment:
Should versions be inherited from `generated-parent`?

Reply:
Yes! You could either decide to do it in this pr or we can do a separate refactor pr along with the same thing in the [module poms](https://github.com/googleapis/gapic-generator-java/blob/autoconfig-gen-draft2/src/main/java/com/google/api/generator/spring/SpringWriter.java#L243-L254). 
btw, I see some unused dependencies in this file that can be removed.

Reply:
Alright! Just removed the dependencies.

Reply:
I'm adding both `spring-boot-starter` and `spring-cloud-gcp-core` to `generated-parent`